### PR TITLE
GearSwap: new wardrobes

### PR DESCRIPTION
**Pull request by Lygre**

### CODE REVIEW

File: addons/GearSwap/helper_functions.lua [hunk 1]
```diff
@@ -606,6 +606,24 @@ function find_usable_item(item_id,bool)
             end
         end
     end
+    if not inventory_index then
+        for i,v in pairs(items.wardrobe3) do
+            if type(v) == 'table' and v.id == item_id and (v.status == 5 or v.status == 0) and (not bool or is_usable_item(v)) then
+                inventory_index = i
+                bag_id = 10
```
Comment (z16) [inline addons/GearSwap/helper_functions.lua]: I think this needs to be `11`, the next one `12`.

Comment (Lygre) [inline addons/GearSwap/helper_functions.lua]: Thanks. Not sure how that happened; it was correct on my other branch.

**Comment by z16** [comment]
I think you were working off of an old branch here. I see your pull request reverts some changes made in PR #1424 ([this block](https://github.com/Windower/Lua/commit/5856bf621d7cbdda93b0869973b0a069eaef8e98#diff-42f8b71f06d2b6227b97b965d0424370L83), for example). Could you please double-check that?

**Comment by Lygre** [comment]
Last commit should be updated from correct branch now.

**Pull request merged**